internal/generator/vector/output/otlp/otlp_all.toml

```diff
@@ -0,0 +1,374 @@
+# Route container, journal, and audit logs separately
+[transforms.output_otel_collector_reroute]
+type = "route"
+inputs = ["pipeline_my_pipeline_viaq_0"]
+route.container = '.log_source == "container"'
+route.node = '.log_source == "node"'
+route.auditd = '.log_source == "auditd"'
+route.kubeapi = '.log_source == "kubeAPI"'
+route.openshiftapi = '.log_source == "openshiftAPI"'
+route.ovn = '.log_source == "ovn"'
+
+# Normalize container log records to OTLP semantic conventions
+[transforms.output_otel_collector_container]
+type = "remap"
+inputs = ["output_otel_collector_reroute.container"]
+source = '''
+  # Create base resource attributes
+  resource.attributes = []
+  resource.attributes = append( resource.attributes,
+      [{"key": "cluster.id", "value": {"stringValue": get!(.,["openshift","cluster_id"])}},
+  	{"key": "openshift.log.source", "value": {"stringValue": .log_source}}]
```

Comment:
Hah, it was there and we decided it needed we needed to group by source.    If we add type here, it will get squashed unless we also group and reduce by matching type.